css/css_test.go

```diff
@@ -155,7 +155,7 @@ func TestCSSInline(t *testing.T) {
 		{"content: \"a\\\nb\";", "content:\"ab\""},
 		{"content: \"a\\\r\nb\\\r\nc\";", "content:\"abc\""},
 		{"content: \"\";", "content:\"\""},
-		{"x: white , white", "x:#fff,#fff"},
+		{"color: white , white", "color:#fff,#fff"},
```

Comment:
Could you add some tests that make sure color-like identifier are preserved in other contexts? (e.g. Apple Pay from original report)

Reply:
Same for KeepCSS2 tests.